Geometry/HcalAlgo/plugins/DDHCalTBZposAlgo.cc

```diff
@@ -35,49 +38,57 @@ void DDHCalTBZposAlgo::initialize(const DDNumericArguments & nArgs,
   dist       = nArgs["Distance"];
   tilt       = nArgs["TiltAngle"];
   copyNumber = int (nArgs["Number"]);
-  LogDebug("HCalGeom") << "DDHCalTBZposAlgo debug: Parameters for position"
-		       << "ing--" << " Eta " << eta << "\tTheta " 
-		       << theta/CLHEP::deg << "\tShifts " << shiftX << ", " 
-		       << shiftY  << " along x, y axes; \tZoffest " << zoffset
-		       << "\tRadial Distance " << dist << "\tTilt angle "
-		       << tilt/CLHEP::deg << "\tcopyNumber " << copyNumber;
+  edm::LogVerbatim("HCalGeom") << "DDHCalTBZposAlgo: Parameters for position"
+			       << "ing--" << " Eta " << eta << "\tTheta " 
+			       << convertRadToDeg(theta) << "\tShifts " 
+			       << shiftX << ", " << shiftY  << " along x, y "
+			       << "axes; \tZoffest " << zoffset
+			       << "\tRadial Distance " << dist 
+			       << "\tTilt angle " << convertRadToDeg(tilt)
+			       << "\tcopyNumber " << copyNumber;
 
   idNameSpace = DDCurrentNamespace::ns();
   childName   = sArgs["ChildName"]; 
-  DDName parentName = parent().name(); 
-  LogDebug("HCalGeom") << "DDHCalTBZposAlgo debug: Parent " << parentName
-		       << "\tChild " << childName << " NameSpace "
-		       << idNameSpace;
+#ifdef EDM_ML_DEBUG
+  edm::LogVerbatim("HCalGeom") << "DDHCalTBZposAlgo: Parent " <<parent().name()
+			       << "\tChild " << childName << " NameSpace "
+			       << idNameSpace;
+#endif
 }
 
 void DDHCalTBZposAlgo::execute(DDCompactView& cpv) {
 
   DDName mother = parent().name();
   DDName child(DDSplit(childName).first, DDSplit(childName).second);
  
-  double thetax = 90.*CLHEP::deg - theta;
+  double thetax = 90._deg - theta;
   double z      = zoffset + dist*tan(thetax);
   double x      = shiftX - shiftY*sin(tilt);
   double y      = shiftY*cos(tilt);
   DDTranslation tran(x,y,z);
   DDRotation rot;
-  double tiltdeg = tilt/CLHEP::deg;
+  double tiltdeg = convertRadToDeg(tilt);
```

Comment:
As I mentioned for #26271 and #26272, we want to eliminate conversion of angles to degrees, except in the debug code.

Reply:
But we had the convention of reusing rotation matrices and the common ones are rotation around z-axis by an angle phi - the convention is Phi where phi is in degrees.

Reply:
There is a utility to create these rotation strings: [formatAsDegrees](https://github.com/cms-sw/cmssw/blob/6383740652b50371bd88dcd809fe8ab28e9c3510/DetectorDescription/Core/interface/DDTypes.h#L36)

If there is anything wrong with this utility or if it is declared in an unsuitable header file, then we should fix it so it can be widely used.